build.gradle.kts

```diff
@@ -10,7 +10,7 @@ plugins {
 
 allprojects {
   group = "uk.gov.justice.service.hmpps"
-  version = "2.2.0"
+  version = "2.2.1-beta"
```

Comment:
is there any point going to beta these days - I don't bother with the gradle plugin